Posts/CV_UserOnlineDocumentation.md

```diff
@@ -0,0 +1,208 @@
+# Google Season of Docs(GSoD) 2020 CircuitVerse Online Documentation
```

Comment:
File should be in `content/posts`, posts is lowercase. 

Should contain metadata in the beginning like this:
```
---
title: "GSoD 2020 CircuitVerse Interactive Book Final Report"
date: 2020-12-02T14:17:50Z
draft: false
author: "Daniel Vicente Lühr Sierra"
type: post
url: "/posts/gsod2020-cvib-final-report-dvls/"
---
```